packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -55,6 +55,7 @@ class SpeedDialAction extends React.Component {
       onClick,
       open,
       tooltipTitle,
+      tooltipPosition,
```

Comment:
`tooltipPlacement`? (Might as well keep it consistent with the Tooltip prop name).